packages/lib-content/src/screens/About/components/Contact.js

```diff
@@ -17,7 +17,19 @@ export default function Contact({ widgetLoaded = false }) {
 
   const handleClick = e => {
     if (widgetLoaded) {
+      const contactButton = document.activeElement
       window.FreshworksWidget('open')
+      setTimeout(() => {
+        const iframe = document.querySelector('iframe#widget-frame')
+        const input = iframe?.contentDocument?.querySelector('input#name')
+        input?.focus()
+        iframe?.contentDocument?.addEventListener('keydown', (event) => {
+          if (event.key === 'Escape') {
+            contactButton.focus()
+            window.FreshworksWidget('close')
+          }
```

Comment:
Focus the contact button, then close the form, otherwise focus jumps back to the `<body>` when the form closes (default browser behaviour when you remove the current `document.activeElement` from the DOM.)